docs/howtos/customizations/metrics/_train_your_own_metric.md

```diff
@@ -0,0 +1,156 @@
+# Train your own metric
+
+LLM as judge metric often makes mistakes and lack alignment with human evaluators. This makes them risky to use as their results cannot be trusted fully. Now, you can fix this using ragas. This simple tutorial notebook showcasing how to train and align any LLM as judge metric using ragas. One can use this to train any LLM based metric in ragas. 
+
+
+## Import required modules
+
+
+```python
+import os
+from datasets import load_dataset
+from ragas import evaluate, EvaluationDataset
+from ragas.metrics import AspectCritic
+
+```
+
+
+Now, sign up for a free account at [app.ragas](https://app.ragas.io) and get your API key.
+Navigate to App tokens -> Create new token. Copy the key and paste it in the below code. Store it safely.
```

Comment:
isn't it better to tell them about the annotation they have to do first and then suggest they use app.ragas.io for this or configure any other solution like labelstudio?

and introduce app.ragas.io in the annotation set

Reply:
the thinking was in this doc the star is the alignment and `.train()` function and app.ragas.io makes it easier but not a requirement